tests/test_serialization.py

```diff
@@ -0,0 +1,257 @@
+import contextlib
```

Comment:
Deleting the old `proto/` folder is maybe a bit more of a discussion since that builds code in https://github.com/coreweave/tensorizer/tree/main/tensors which is currently used in software external to this repo.

Reply:
For context I'm currently working on a small reorganization of these files that will keep `proto/` and `tensors/` in the source distributions of this repository (since they are the sources that generate `tensorizer/tensors_pb2.py`) but not in the wheel distributions, and organize the other stray files in the root directory.
The `protobuf.py` and `tensorizer/tensors_pb2.py` code is a bit tricky, because its binary format is likely out of date but still sort of functional, yet needs to be included. We might end up bringing it more up to date separately after the main release since it isn't even documented.

Reply:
We should not be deleting the `proto/` directory. It serves a purpose outside of this package.